### ISSUE TITLE
Outdated pypi package

### ISSUE THREAD

**Issue by cumason123** [cloudevents/sdk-python#73]
## Expected Behavior
Following code should work using the [pypi](https://pypi.org/project/cloudevents/) package.
```
pip install cloudevents
python
>>> from cloudevents.sdk.http import CloudEvent
```

## Actual Behavior
```
pip install cloudevents
python
>>> from cloudevents.sdk.http import CloudEvent
```
Throws error because module http does not exist due to pypi being outdated. 

Ultimately i'm asking if everyone thinks branch v1.0.0-dev should be considered stable now, and if not what else can be done.

**Comment by cumason123** [comment]
@evankanderson @di I would like to publish v1.0.0-dev this week if possible. Any feedback on additional issues we should resolve before publishing would be appreciated!

**Comment by di** [comment]
@cumason123 Should we consider publishing a [pre-release](https://packaging.python.org/guides/distributing-packages-using-setuptools/#pre-release-versioning) on PyPI first, to allow consumers to test before publishing 1.0.0?

**Comment by grant** [comment]
Sounds good. Things seem pretty stable and Curtis has added a lot of improvements with PRs.
I think we can go with a normal release unless there's a good reason for a prerelease? Not sure which consumers we'd expect to get feedback from within some timeframe.

**Comment by grant** [comment]
- Regular release should be fine. (Not pre-release)
- GitHub Action – Release on merge (https://github.com/pypa/gh-action-pypi-publish)
  - Previously had `release.sh` w/ env var with version
  - AI: @di to set secret on GitHub
  - Ping @di for PR review

**Comment by evankanderson** [comment]
My only question/comment would be if the current API (as `cloudevents.sdk.???` with `base`, `http`, etc) is the API that you want to publish for v1.0. (You can always ship a v2.0, and I think the Go API has already done so, so perhaps having something published as v1.0 and then cleaning up from there makes sense.)

**Comment by cumason123** [comment]
I'm working on a follow-up PR to reduce the import schema from `cloudevents.sdk.???` to `cloudevents.???`. Issue creations are starting to die down so I'm hoping the cloudevents community can provide more feedback before I update the package, but you're correct in stating we can always adjust through later minor version updates.
